backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -68,6 +72,39 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
                         .body(response.bodyToMono(Object.class))));
     }
 
+    @GetMapping("/users/{userId}/role-mappings/clients/{clientGuid}")
```

Comment:
Missing integration tests...

Reply:
Added some tests and fixed the NPE (per teams discussion) 
Didn't add tests for 'authorized' clients because if we make up the clientId in the current state it will just gives a 401. Once the http response code refactor is implemented I can make more tests.